api/src/domain/license-status/searchLicenseStatusFactory.ts

```diff
@@ -1,18 +1,19 @@
 import { SearchLicenseStatus } from "@domain/types";
 
-export const searchLicenseStatusFactory = (
-  legacySearchLicenseStatus: SearchLicenseStatus,
-  newSearchLicenseStatus: SearchLicenseStatus
-) => {
+export const searchLicenseStatusFactory = (legacy: SearchLicenseStatus, updated: SearchLicenseStatus) => {
   return (licenseType: string): SearchLicenseStatus => {
-    const useNewSearchLicenseStatus = (): boolean =>
-      licenseType === "Public Movers and Warehousemen" &&
-      process.env.FEATURE_DYNAMICS_PUBLIC_MOVERS === "true";
+    const FeatureFlags: Record<string, boolean> = {
+      publicMovers: process.env.FEATURE_DYNAMICS_PUBLIC_MOVERS === "true",
+      healthCareServices: process.env.FEATURE_DYNAMICS_HEALTH_CARE_SERVICES === "true",
+    };
 
-    if (useNewSearchLicenseStatus()) {
-      return newSearchLicenseStatus;
-    }
+    const LicenseTypesWithUpdatedSearch: Record<string, boolean> = {
```

Comment:
[dust] Should this be something like `licenseTypesUsingUpdatedSearch` to better describe the reason for the boolean?

Also any particular reason these variables are not camelcase?

Reply:
good catch, it should be camel case